### PR TITLE
[Dungeon] Razorfen Kraul

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588337024794996000.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` = 4514;
+UPDATE `creature_template` SET `mindmg` = 63, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` = 4511;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 193, `maxdmg` = 249, `DamageModifier` = 1.02 WHERE `entry` = 4842;
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 208, `maxdmg` = 266, `DamageModifier` = 1.02 WHERE `entry` = 4425;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.01 WHERE `entry` = 4424;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.01 WHERE `entry` = 4428;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337024794996000.sql:47]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.01 WHERE `entry` IN (4424, 4428);
```

File: data/sql/updates/pending_db_world/rev_1588337024794996000.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+UPDATE `creature_template` SET `mindmg` = 68, `maxdmg` = 92, `DamageModifier` = 1.03 WHERE `entry` = 4539;
+UPDATE `creature_template` SET `mindmg` = 68, `maxdmg` = 92, `DamageModifier` = 1.03 WHERE `entry` = 4538;
+UPDATE `creature_template` SET `mindmg` = 58, `maxdmg` = 77, `DamageModifier` = 1.03 WHERE `entry` = 4427;
+UPDATE `creature_template` SET `mindmg` = 56, `maxdmg` = 77, `DamageModifier` = 1.03 WHERE `entry` = 4519;
+UPDATE `creature_template` SET `mindmg` = 38, `maxdmg` = 50, `DamageModifier` = 1.03 WHERE `entry` = 4625;
+UPDATE `creature_template` SET `mindmg` = 68, `maxdmg` = 92, `DamageModifier` = 1.03 WHERE `entry` = 4541;
+UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` = 4437;
+UPDATE `creature_template` SET `mindmg` = 56, `maxdmg` = 75, `DamageModifier` = 1.03 WHERE `entry` = 4517;
+UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` = 4514;
+UPDATE `creature_template` SET `mindmg` = 63, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` = 4511;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337024794996000.sql:39]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` IN (4530, 4436, 4437, 4514);
UPDATE `creature_template` SET `mindmg` = 57, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 4535;
UPDATE `creature_template` SET `mindmg` = 63, `maxdmg` = 88, `DamageModifier` = 1.03 WHERE `entry` IN (4435, 4511);
UPDATE `creature_template` SET `mindmg` = 65, `maxdmg` = 92, `DamageModifier` = 1.03 WHERE `entry` IN (4442, 4438, 6035);
UPDATE `creature_template` SET `mindmg` = 43, `maxdmg` = 62, `DamageModifier` = 1.03 WHERE `entry` = 4516;
UPDATE `creature_template` SET `mindmg` = 56, `maxdmg` = 77, `DamageModifier` = 1.03 WHERE `entry` IN (4523, 4522, 4440, 4518, 4519);
UPDATE `creature_template` SET `mindmg` = 56, `maxdmg` = 75, `DamageModifier` = 1.03 WHERE `entry` IN (4520, 4517);
UPDATE `creature_template` SET `mindmg` = 54, `maxdmg` = 80, `DamageModifier` = 1.03 WHERE `entry` = 4528;
UPDATE `creature_template` SET `mindmg` = 68, `maxdmg` = 92, `DamageModifier` = 1.03 WHERE `entry` IN (4512, 4623, 4539, 4538, 4541);
UPDATE `creature_template` SET `mindmg` = 68, `maxdmg` = 92, `minrangedmg` = 48, `maxrangedmg` = 71, `DamageModifier` = 1.03 WHERE `entry` = 4532;
UPDATE `creature_template` SET `mindmg` = 36, `maxdmg` = 48, `DamageModifier` = 1.03 WHERE `entry` = 4534;
UPDATE `creature_template` SET `mindmg` = 117, `maxdmg` = 151, `DamageModifier` = 1.03 WHERE `entry` = 6021;
UPDATE `creature_template` SET `mindmg` = 43, `maxdmg` = 60, `DamageModifier` = 1.03 WHERE `entry` = 4526;
UPDATE `creature_template` SET `mindmg` = 58, `maxdmg` = 77, `DamageModifier` = 1.03 WHERE `entry` IN (4525, 4427);
UPDATE `creature_template` SET `mindmg` = 38, `maxdmg` = 50, `DamageModifier` = 1.03 WHERE `entry` = 4625;
```

File: data/sql/updates/pending_db_world/rev_1588337024794996000.sql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 193, `maxdmg` = 249, `DamageModifier` = 1.02 WHERE `entry` = 4842;
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 208, `maxdmg` = 266, `DamageModifier` = 1.02 WHERE `entry` = 4425;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.01 WHERE `entry` IN (4424, 4428);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 193, `maxdmg` = 249, `DamageModifier` = 1.01 WHERE `entry` = 4420;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 160, `maxdmg` = 205, `DamageModifier` = 1.01 WHERE `entry` = 4422;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 185, `maxdmg` = 239, `DamageModifier` = 1.01 WHERE `entry` = 4421;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 6168;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337024794996000.sql:34]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 146, `maxdmg` = 189, `DamageModifier` = 1.01 WHERE `entry` IN (4424, 4428);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 193, `maxdmg` = 249, `DamageModifier` = 1.01 WHERE `entry` = 4420;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 160, `maxdmg` = 205, `DamageModifier` = 1.01 WHERE `entry` = 4422;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 185, `maxdmg` = 239, `DamageModifier` = 1.01 WHERE `entry` = 4421;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 134, `maxdmg` = 173, `DamageModifier` = 1.01 WHERE `entry` = 6168;
```

**Comment by Geekena** [comment]
damage and spells are ok
only higher damage is from agem boss when he spawn pig to fight with him

**Comment by Knindzagxg** [comment]
Thanks a lot for testing Geekena.

On topic - Yes, it's supposed to be higher damage when Boar appears.
Tanks are required to pick them quickly and others should switch quick to reduce further incoming damage.

**Comment by Kitzunu** [comment]
ty for the PR :)

**Pull request merged**